unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -435,16 +437,26 @@ private static void equivalencesLine(String line, ParsePosition pp) throws Parse
             ++testFailureCount;
             printErrorLine("Test Failure", Side.START, testFailureCount);
         }
+        final List<String> errorMessage = new ArrayList<>();
```

Comment:
Nit/optional: It seems like a container should have a plural name. Maybe errorMessageLines?